.ci/jobs/deploy-package-registry-pipeline.yml

```diff
@@ -0,0 +1,24 @@
+---
+- job:
+    name: deploy-package-registry
```

Comment:
Nit. Nothing is actually deployed here, but released, or tagged.

Reply:
Yep, what name do you think it could be the best one?

Reply:
`release-package-registry-distribution`